### DISCUSSION TITLE
Can't find variable: process, if I run npm build and npm start

### DISCUSSION THREAD

**Discussion by ahoiroman** [nuxt/nuxt#10456] [Questions (Nuxt 2)] [answered]
Hello,

I am trying to deploy my nuxt3-rc2 application to a Forge-provisioned server and followed these steps: https://web.staging.happi.li

Now if I try to access the nuxt instance, I am getting this error:

```
500
Can't find variable: process
```

Unfortunately I don't see something like a stack trace, but I am wondering, why Nuxt is throwing this error, as I am no longer using `process.env` in my app.

<img width="745" alt="Bildschirmfoto 2022-05-18 um 23 37 27@2x" src="https://user-images.githubusercontent.com/23051150/169160102-2cf06942-5590-47e5-ac79-cb95eaed2bce.png">

**Comment by gplusdotgr** [comment, marked as answer]
@ahoiroman there's a separate github repo for Nuxt3, https://github.com/nuxt/framework/discussions , you might have better luck there. 

I guess the stack trace is not shown as this is a production build, what happens when you try a production build locally ? would be easier to debug. ( try starting the server with the node --inspect flag)

**Comment by ahoiroman** [comment]
That was very helpful, thanks!

Turned out it was the i18n-module.

**Reply by gplusdotgr** [reply]
always happy to help
